content/en/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -229,7 +229,7 @@ can happen, according to:
 
 ## Pod disruption conditions {#pod-disruption-conditions}
```

Comment:
It should link to pod lifecycle somewhere.

Reply:
We have ended up making the change without the feature gate, so not sure the note is relevant here. I add the note from the page about Jobs, where I find it more relevant, because it fixes an issue there.